tests/paz/pipelines/pix2pose_metric_test.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import cv2
+import os
+from tensorflow.keras.utils import get_file
+from paz.backend.image import load_image
+from paz.backend.camera import Camera
+from paz.pipelines import PIX2POSEPowerDrill
+from paz.backend.groups.quaternion import quaternion_to_rotation_matrix, rotation_matrix_to_quaternion
+import re
+from scipy import spatial
+from scipy.spatial import distance
+import numpy as np
+from paz.abstract.messages import Pose6D
+from examples.pix2pose.eval import compute_ADI, compute_ADD, transform_points3D, get_translation, get_quaternion
+
+
+@pytest.fixture
+def image_with_powerdrill():
+    URL = ('https://github.com/oarriaga/altamira-data/releases/download'
+           '/v0.9.1/image_with_YCB_objects.jpg')
+    filename = os.path.basename(URL)
+    fullpath = get_file(filename, URL, cache_subdir='paz/tests')
+    image = load_image(fullpath)
+    return image
+
+
+@pytest.fixture
+def get_pose():
```

Comment:
So fixtures are strange. You should define the function name as the argument to your tests. For example.

def get_pose -> def true_pose

Reply:
resolved